Gemfile

```diff
@@ -24,6 +24,7 @@ gem 'values'
 gem 'rack-cors', :require => 'rack/cors'
 gem 'exception_notification'
 gem 'slack-notifier'
+gem 'rack-attack'
```

Comment:
it is best practice that we dont yet use here to lock gem versions in the gemfile. 
`gem 'rack-attack' "~> 5.0.1"`